### PR TITLE
docs(readme): add types

### PR DESCRIPTION
**Pull request by harish-sethuraman**
This PR contains a:

- [ ] **bugfix**
- [ ] new **feature**
- [ ] **code refactor**
- [ ] **test update** 
- [ ] **typo fix**
- [x] **metadata update**

### Motivation / Use-Case

Added types

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 # json-minimizer-webpack-plugin
 
-This plugin uses [JSON.stringify()](https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify) to minify your JSON.
+This plugin uses [JSON.stringify()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify) to minify your JSON.
```
Comment (harish-sethuraman) [inline README.md:17]: Seems like these were links to Russian translation of stringify so changed it to english as well

File: README.md [hunk 2]
```diff
@@ -150,10 +164,18 @@ module.exports = {
...
 ### `minimizerOptions`
 
-Type: `Object`
+Type:
+
+```ts
+type minimizerOptions = {
+  space?: string | number;
+  replacer?: null | Function;
+};
```
Comment (snitin315) [inline README.md:173]: As per our schema - https://github.com/webpack-contrib/json-minimizer-webpack-plugin/blob/8a65b2cb7b6c9fb9745872e2b2e36d79c508f0ea/src/options.json#L76-L98
```suggestion
type minimizerOptions = {
  space?: null | string | number;
  replacer?: null | Function | Array<string | number>;
};
```

Comment (harish-sethuraman) [reply README.md:173]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#the_replacer_parameter
Im guessing there are more types too? like boolean and Symbol as well?

Comment (snitin315) [reply README.md:173]: Hmm, looks like we don't support them at the moment.

/cc @alexander-akait

**Review by alexander-akait — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (alexander-akait) [reply README.md:173]: Oh, let's add test case for this

Comment (harish-sethuraman) [reply README.md]: @snitin315  I've changed the type as per you have mentioned. We shall probably add boolean and symbol along with tests when we add them? Thoughts?

**Comment by harish-sethuraman** [comment]
@alexander-akait Requesting review

**Review by alexander-akait — APPROVED** [review]

**Comment by alexander-akait** [comment]
Thanks

**Pull request merged**